web/src/components/AlbumTableSkeleton.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <v-skeleton-loader ref="skeleton" :boilerplate="false" type="table" :tile="false" class="mx-auto"></v-skeleton-loader>
+</template>
+
+<script>
+export default {
+  name: 'skeleton.album.table',
```

Comment:
I think the convention for component names in Vue is `AlbumTableSkeleton` (i.e. match the filename)